src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLogger.cs

```diff
@@ -78,6 +78,11 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                         if (logger.BindProperty(property.Key.Substring(1), property.Value, true, out var destructured))
                             properties.Add(destructured);
                     }
+                    else if (property.Key.StartsWith("$"))
+                    {
+                        if (logger.BindProperty(property.Key.Substring(1), property.Value?.ToString() ?? "null", true, out var stringified))
```

Comment:
Substituting in the text `null` will be inconsistent with Serilog here, I think. We should just pass null if `property.Value` is null.